From 2fd3604ad6a9594b9dd8b0f1529f4f4589fe831f Mon Sep 17 00:00:00 2001
From: Carl Schwan <carl@carlschwan.eu>
Date: Tue, 19 Sep 2023 17:04:36 +0200
Subject: [PATCH] Revert "Revert "Fix race condition when building""

This reverts commit d3f20b3dab530065f9cff9d73e4ca1076af55a29.

(cherry picked from commit f9f262b9289f7b1c34e55205fa0c5cde915bc652)
---
 resources/google-groupware/CMakeLists.txt     | 7 ++++---
 resources/google-groupware/googleresource.cpp | 1 -
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/resources/google-groupware/CMakeLists.txt b/resources/google-groupware/CMakeLists.txt
index 2f2b46aded..0ac6f2db8e 100644
--- a/resources/google-groupware/CMakeLists.txt
+++ b/resources/google-groupware/CMakeLists.txt
@@ -67,9 +67,6 @@ set(googleresource_SRCS
     ${googleresource_common_SRCS}
 )
 
-ki18n_wrap_ui(googleresource_SRCS googlesettingswidget.ui)
-
-
 add_executable(akonadi_google_resource ${googleresource_SRCS})
 if (COMPILE_WITH_UNITY_CMAKE_SUPPORT)
     set_target_properties(akonadi_google_resource PROPERTIES UNITY_BUILD ON)
@@ -126,6 +123,7 @@ install(
 kcoreaddons_add_plugin(googleconfig
     INSTALL_NAMESPACE "pim${QT_MAJOR_VERSION}/akonadi/config"
 )
+
 target_sources(googleconfig
 PRIVATE
     googleconfig.cpp
@@ -133,6 +131,9 @@ PRIVATE
     googlesettingswidget.h
     ${googleresource_common_SRCS}
 )
+
+ki18n_wrap_ui(googleconfig googlesettingswidget.ui)
+
 target_link_libraries(googleconfig
     KPim${KF_MAJOR_VERSION}::AkonadiCore
     KPim${KF_MAJOR_VERSION}::AkonadiCalendar
diff --git a/resources/google-groupware/googleresource.cpp b/resources/google-groupware/googleresource.cpp
index 47a7e0c8d5..d51c1e4a7c 100644
--- a/resources/google-groupware/googleresource.cpp
+++ b/resources/google-groupware/googleresource.cpp
@@ -11,7 +11,6 @@
 #include "googleresourcestate.h"
 #include "googlescopes.h"
 #include "googlesettings.h"
-#include "googlesettingswidget.h"
 #include "settingsadaptor.h"
 
 #include "personhandler.h"
-- 
GitLab

